<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateWindow</class>
 <widget class="QMainWindow" name="UpdateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>thePackage Updater</string>
  </property>
  <property name="windowIcon">
   <iconset theme="system-software-update"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>System Updates</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>System updates keep your computer running fast and safe.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="updatesAvaliable"/>
    </item>
    <item>
     <widget class="QTextBrowser" name="pacmanOutput">
      <property name="font">
       <font>
        <family>Noto Mono</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="powerWarning">
      <property name="text">
       <string>You should plug your PC in to power before you update.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="maximum">
       <number>0</number>
      </property>
      <property name="value">
       <number>-1</number>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset theme="window-close"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Install System Updates</string>
        </property>
        <property name="icon">
         <iconset theme="system-software-update"/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
